viz.yaml

```diff
@@ -119,7 +119,7 @@ fetch:
     id: histWaterUseAWUDS-data
     location: cache/awuds.zip
     fetcher: sciencebase
-    remoteItemId: "5845ffaae4b04fc80e523664"
+    remoteItemId: "584aa8b4e4b07e29c706dd57"
```

Comment:
So, if you move your file over to the private sb repo, then change this back (I think?)